WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewController.swift

```diff
@@ -38,30 +38,38 @@ final class ShippingLabelAddressFormViewController: UIViewController {
             guard let self = self else { return }
             ServiceLocator.analytics.track(.shippingLabelEditAddressContactCustomerButtonTapped)
 
-            let actionSheet = UIAlertController(title: nil, message: Localization.contactActionLabel, preferredStyle: .actionSheet)
-            actionSheet.view.tintColor = .text
-
-            actionSheet.addCancelActionWithTitle(Localization.contactActionCancel)
-            if let email = email, email.isNotEmpty {
-                actionSheet.addDefaultActionWithTitle(Localization.contactActionEmail) { _ in
-                    self.sendEmail(to: email)
-                }
-            }
-            if let phoneNumber = phone, phoneNumber.isNotEmpty {
-                actionSheet.addDefaultActionWithTitle(Localization.contactActionCall) { _ in
-                    if PhoneHelper.callPhoneNumber(phone: phoneNumber) == false {
-                        self.displayErrorNotice(title: Localization.phoneNumberErrorNotice)
+            if PhoneHelper.canCallPhoneNumber(phone: "123456789") == false
```

Comment:
This has a (fake) string literal in it. Do you mean to check `phone` there?

```suggestion
            if PhoneHelper.canCallPhoneNumber(phone: phone) == false
```